pkgs/development/tools/github-commenter/default.nix

```diff
@@ -20,6 +20,6 @@ buildGoPackage rec {
     license = licenses.asl20;
     homepage = "https://github.com/cloudposse/github-commenter";
     maintainers = [ maintainers.mmahut ];
-    platforms = platforms.linux;
+    platforms = with platforms; linux ++ darwin;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
yeah, I've already done that in my other contributions. I'll upgrade here too.